packages/manager/core/app/src/components/Dashboard/DashboardTile.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { Tile } from '@ovh-ux/manager-themes';
+
+import { TileTypesEnum, DashboardTile as DashboardTileType } from '.';
+import DashboardTileLoading from './DashboardTileLoading';
+import DashboardTileList from './DashboardTileList';
+
+export type DashboardTileProps = {
+  tile: DashboardTileType;
+};
+
+export default function DashboardTile({
+  tile,
+}: DashboardTileProps): JSX.Element {
+  const [isLoading, setIsLoading] = useState(true);
+  const [data, setData] = useState<unknown>();
```

Comment:
if all of your tile components are using some data but you don't know in advance what the data will be, it can be good to use templates instead of unknown :
```ts
DashboardTile<T>
const [data, setData] = useState<T>();
```
same for other components :
```ts
DashboardTileList<T> 
```
and for example if you know that the data will always have a description you can then provides some typing by using :
```ts
interface DashboardTileData {
  description: string;
};
DashboardTileList<T extends DashboardTileData> 
```
that way your component will work for abstract data but you can add your own constraints (for example i want my component to work with any object that as a "description" attribute)